test/index.js

```diff
@@ -1,16 +1,17 @@
-const bip21 = require('../')
-const fixtures = require('./fixtures')
-const tape = require('tape')
+import { encode, decode as _decode } from '../index.js'
```

Comment:
why `_decode`? (I'm curious)

Reply:
changed! (VSCode did it when updating imports)